### PR TITLE
added android tv icon support

### PR DESCRIPTION
**Pull request by HADeveloper**

### CODE REVIEW

File: packages/config-tv/src/types.ts [hunk 1]
```diff
@@ -70,6 +70,13 @@ export type ConfigData = {
    * Android resource drawable directories.
    */
   androidTVBanner?: string;
+  /**
+   * If set, this should be a path to an existing PNG file appropriate for an Ancroid TV icon image.
```
Comment (douglowder) [inline packages/config-tv/src/types.ts]:
```suggestion
   * If set, this should be a path to an existing PNG file appropriate for an Android TV icon image.
```

File: packages/config-tv/src/types.ts [hunk 2]
```diff
@@ -70,6 +70,13 @@ export type ConfigData = {
...
+   * See https://developer.android.com/design/ui/tv/guides/system/tv-app-icon-guidelines#launcher-icon
+   * The Android manifest will be modified to reference this image and the image will be copied into
```
Comment (douglowder) [inline packages/config-tv/src/types.ts:77]:
```suggestion
   * The icon image will not be resized.
   * The Android manifest will be modified to reference this image and the image will be copied into
```

**Review by douglowder — CHANGES_REQUESTED** [review]
Looks good! I added a couple of suggested changes in the Typescript doc comments.

It would also be great if you could add a description of the new plugin property in the README!

**Comment by HADeveloper** [comment]
@douglowder I made the requested changes. Let me know if anything else needs to be updated. Thanks!

**Review by douglowder — APPROVED** [review]

**Pull request merged**